<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditAudit</class>
 <widget class="QDialog" name="DialogEditAudit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Dépenses de fonctionnement</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Recettes de fonctionnement</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tableViewDF">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QTableView" name="tableViewRF"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Dépenses d'investissement</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Recettes d'investissement</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTableView" name="tableViewDI"/>
   </item>
   <item row="3" column="2">
    <widget class="QTableView" name="tableViewRI"/>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboListAudits"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
